src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -2788,12 +2802,12 @@ public enum LegendsAwakened implements LogicCardInfo {
             text "Once during your turn (before your attack), when you attach a [M] Energy card from your hand to Beldum (excluding effects of attacks or Poké-Powers), you may search your deck for Beldum and put it onto your Bench. Shuffle your deck afterward. This power can't be used if Beldum is affected by a Special Condition."
             delayedA {
               before ATTACH_ENERGY, {
-                if (ef.reason == PLAY_FROM_HAND && ef.card.asEnergyCard().containsType(W) && bg.currentTurn == self.owner && ef.resolvedTarget == self && self.noSPC()) {
+                if (ef.reason == PLAY_FROM_HAND && ef.card.asEnergyCard().containsType(M) && bg.currentTurn == self.owner && ef.resolvedTarget == self && self.noSPC() && confirm("Use Metal Chain?")) {
                   checkLastTurn()
                   assert my.bench.notFull : "Bench full"
                   assert deck.notEmpty : "Deck is empty"
```

Comment:
These should be in the if statement. No reason to ask the player "want to use this ability?" "too bad you can't"